frontend/src/v5/store/revisions/revisions.redux.ts

```diff
@@ -30,6 +31,8 @@ export const { Types: RevisionsTypes, Creators: RevisionsActions } = createActio
 	createRevision: ['teamspace', 'projectId', 'uploadId', 'body'],
 	setUploadComplete: ['uploadId', 'isComplete', 'errorMessage'],
 	setUploadProgress: ['uploadId', 'progress'],
+	fetchRevisionStatsSuccess: ['containerId', 'data'],
+	revisionProcessingSuccess: ['containerId', 'revision'],
```

Comment:
Can we change this one to newRevisionSuccess or addedRevisionSuccess?

Reply:
The problem is I don't want it to get confused with the createRevision or the setUploadComplete actions. I thought it might be best to specify that this is for when the backend processing is complete, and not when the initial uploading is complete

Reply:
Mmh ok, lets leave it like this for now, but I think we have specialized too much the actions, therefore we have too many of them